scripts/diagnostics.txt

```diff
@@ -725,6 +725,7 @@ error NTResolveArgModify "resolution function for nettype '{}' cannot modify its
 warning enum-range EnumRangeLiteral "enum element ranges must be integer literals"
 
 subsystem Lookup
+warning duplicate-definition DuplicateDefinition "duplicate definition of '{}'"
```

Comment:
Can you move this to the bottom of this section (the convention is to put warnings at the end of their respective sections) and also add it to the `default` group of warnings that are on by default down at the bottom of the file?

Reply:
Also add documentation and an example to warning_docs.txt. When the docs are built it will run the slang tool on your snippet and ensure that it produces the specified warning, and then include the output in the docs. See https://sv-lang.com/warning-ref.html for the current output.